lib/portage/tests/resolver/ResolverPlayground.py

```diff
@@ -129,8 +129,11 @@ def __init__(
         """
 
         self.debug = debug
-        if eprefix is None:
-            self.eprefix = normalize_path(tempfile.mkdtemp())
+        if True:
```

Comment:
Is this just to avoid a change in indentation?

Reply:
Yeah, we can go ahead and dedent the whole thing.